critiquebrainz/data/model/user.py

```diff
@@ -95,9 +95,9 @@ def reviews(self):
     def avatar(self):
         """Link to user's avatar image."""
         if self.show_gravatar and self.email:
-            return "https://gravatar.com/avatar/" + hashlib.md5(self.email.encode("ascii")).hexdigest() + "?d=mm&r=pg"
+            return "https://gravatar.com/avatar/" + hashlib.md5(self.email.encode("ascii")).hexdigest() + "?d=identicon&r=pg"
         else:
-            return "https://gravatar.com/avatar/placeholder?d=mm"
+            return "https://gravatar.com/avatar/placeholder?d=identicon"
```

Comment:
Well, the hack does seem to generate a gravatar that I know isn't the default one

![screenshot from 2016-12-09 02-50-13](https://cloud.githubusercontent.com/assets/12889549/21028247/3b9c4c9e-bdba-11e6-8812-4fac1627d80b.png)